packages/data-stores/src/reader/queries/use-read-feed-search-query.ts

```diff
@@ -0,0 +1,71 @@
+import { Railcar } from '@automattic/calypso-analytics';
+import { useInfiniteQuery } from '@tanstack/react-query';
+import wpcomRequest from 'wpcom-proxy-request';
+
+export enum FeedSort {
```

Comment:
I remembered somewhere in the back of my head that TypeScript enums aren't very popular, because they can act strange. I watched [this video](https://www.youtube.com/watch?v=jjMbPt_H3RQ) and when you assign string values to them, they're okay 👍